.github/workflows/manypkg.yml

```
@@ -0,0 +1,46 @@
+name: Manypkg Check
```

This name doesn't tell too much to someone who is not familiar with `manypkg`. Maybe something like this? 👇 Perhaps should also change the filename accordingly.

```suggestion
name: Monorepo integrity
```

---

Done, file has been renamed!